plugins/modules/source_control/gitlab/gitlab_runner.py

```diff
@@ -332,6 +342,7 @@ def main():
         maximum_timeout=dict(type='int', default=3600),
         registration_token=dict(type='str', no_log=True),
         project=dict(type='str'),
+        group=dict(type='str'),
         state=dict(type='str', default="present", choices=["absent", "present"]),
```

Comment:
Shouldn't you prevent that `project` and `group` are used together?

Reply:
@felixfontein 
I added a check at L385, but I've just seen the `mutually_exclusive` parameter of `AnsibleModule.__init__` function.

I suppose it does the job ?

Reply:
Using the `mutually_exclusive` mechanism for this is preferred over adding an `if` statement.

Reply:
Thanks, I replaced the if check with a `mutually_exclusive` entry.